config.yaml

```diff
@@ -10,7 +10,7 @@ livePreviewConfiguration:
   setupCommand: ":"
 siteStream:
   id: site-stream
-  entityId: site
+  entityId: site-YEXT_PUBLIC_SITE_ID
```

Comment:
Did you vet this way with Aaron already? Also, does this work? How does the build system know this is an env var and not just a string?